pylabrobot/resources/carrier.py

```diff
@@ -200,6 +200,22 @@ def __init__(
       sites,category=category, model=model)
 
 
+class MFXCarrier(Carrier):
+  """ Base class for multiflex carriers 
+  (i.e. carriers with mixed-use and/or specialized sites). """
+  def __init__(
+    self,
+    name: str,
+    size_x: float,
+    size_y: float,
+    size_z: float,
+    sites: Optional[List[CarrierSite]] = None,
+    category="plate_carrier",
```

Comment:
Should this be mfx_carrier or similar?

Reply:
Yes, you are absolutely right. Missed that change. Implemented it in the last update.